platforms/chibios/platform.mk

```
@@ -402,6 +402,17 @@ ifeq ($(strip $(MCU)), risc-v)
                -mabi=$(MCU_ABI) \
                -mcmodel=$(MCU_CMODEL) \
                -mstrict-align
+
+    # Deal with different arch revisions and gcc renaming them
+    ifneq ($(shell echo 'int main() { asm("csrc 0x300,8"); return 0; }' | $(TOOLCHAIN)gcc $(MCUFLAGS) $(TOOLCHAIN_CFLAGS) -x c -o /dev/null - 2>/dev/null >/dev/null; echo $$?),0)
+        MCUFLAGS = -march=$(MCU_ARCH)_zicsr_zifencei \
```

```suggestion
        MCUFLAGS = -march=$(MCU_ARCH)_zicsr \
```

Might be enough.
Tested this with GCC v11.3/12.3/13.2

---

Yep, thanks.